source/basic.tex

```diff
@@ -3899,6 +3899,8 @@
 in~\ref{conv.prom}.}
 A synonym for integral type is
 \indextext{integer~type}%
+\indextext{complement!ones'}%
+\indextext{complement!two's}%
```

Comment:
Yes, since it appears that you are indexing the various possible representations of signed integers here. So it would be strange to index only some of them, don't you think?